assignments/array-methods.js

```diff
@@ -56,28 +56,76 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(item) {
+    fullName.push(`${item.first_name} ${item.last_name}`);
+});
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(function(item) {
+    allCaps.push(item.first_name.toUpperCase());
```

Comment:
It is not quite how we would use a  `map` function. The callback to the map function should return a transformed element from the original element and the `map` function itself returns a new array filled with the transformed elements. This is how we would typically use it

```js
 let allCaps = runners.map(function (runner) {
  return runner.first_name.toUpperCase();
});
```